types/guacamole-client/lib/Client.d.ts

```diff
@@ -54,7 +54,7 @@ export class Client {
     /**
      * @param tunnel The tunnel to use to send and receive Guacamole instructions.
      */
-    constructor(tunnel: WebSocketTunnel);
+    constructor(tunnel: Tunnel);
```

Comment:
I mistakenly put the concrete type here instead of the parent type: https://github.com/apache/guacamole-client/blob/754e9649f1fa0ba225ee42b56ded64bc283d17df/guacamole-common-js/src/main/webapp/modules/Client.js#L28